plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/presentation/DependencyProvider.java

```diff
@@ -187,21 +194,29 @@
     private Shell dialogShell;
     private String OS_TYPE;
     private String[] connectionURLArray;
+    private String serverName = "server";
     private static IDeveloperStudioLog log = Logger.getLog(Activator.PLUGIN_ID);
 
-    public DependencyProvider(Shell parent, int style) {
+    public DependencyProvider(Shell parent, int style, String activeProjectName, String[] dataBaseTypes) {
         super(parent, style);
         eclipseWorkspace = ResourcesPlugin.getWorkspace().getRoot().getLocation().toString();
         IEditorPart editorPart = PlatformUI.getWorkbench().getActiveWorkbenchWindow()
                 .getActivePage().getActiveEditor();
-        if (editorPart != null) {
-            IFileEditorInput input = (IFileEditorInput) editorPart.getEditorInput();
-            IFile file = input.getFile();
-            IProject activeProject = file.getProject();
-            activeProjectName = activeProject.getName();
+        if (dataBaseTypes !=null) {
```

Comment:
Fix formatting issues in #205, #206, #208, and #210

Reply:
Fixed